### PR TITLE
Provide placeholder values for location and organisation filters

### PR DESCRIPTION
**Pull request by TobyRet**

### CODE REVIEW

File: app/assets/stylesheets/govuk-overrides.scss [hunk 1]
```diff
@@ -49,3 +49,11 @@
     }
   }
 }
+
+// ______________________________________________________
+
+// Filters
+
+.autocomplete__input {
```
Comment (TobyRet) [inline app/assets/stylesheets/govuk-overrides.scss]: I made the assumption that is was the most appropriate place to store the override?

**Comment by TobyRet** [comment]
[When you’ve entered a filter, the value should also be displayed in the filter text field after you click "Apply filters"](https://trello.com/c/JVnsyLeJ/370-when-youve-entered-a-filter-the-value-should-also-be-displayed-in-the-filter-text-field-after-you-click-apply-filters)

File: app/views/search/_filters.html.erb [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
   Enter geographical area
 </label>
-<div id="location" name="location" data-default="<%= @location %>" class="dgu-filters__filter"></div>
+<div style="text-indent: 2px;" id="location" name="location" placeholder="<%= @location %>" data-default="<%= @location %>" class="dgu-filters__filter"></div>
 
 <label for="publisher" class="form-label">
   Enter publisher
 </label>
-<div id="publisher" name="publisher" data-default="<%= @organisation %>" class="dgu-filters__filter"></div>
+<div id="publisher" name="publisher" placeholder="<%= @organisation %>" data-default="<%= @organisation %>" class="dgu-filters__filter"></div>
```
Comment (govuklaurence) [inline app/views/search/_filters.html.erb]: @TobyRet don't set `placeholder`, set `value` if possible

**Comment by TobyRet** [comment]
@maxf will be replacing functionality with Twitter Typeahead

**Pull request closed without merging**